interpreter/interpreter_test.go

```diff
@@ -88,6 +89,19 @@ func TestInterpreter_ComprehensionExpr(t *testing.T) {
 	}
 }
 
+func TestInterpreter_InList(t *testing.T) {
+	parsed, err := parser.ParseText("1 in [1, 2, 3]")
+	if len(err.GetErrors()) != 0 {
+		t.Error(err)
+	}
+	prg := NewProgram(parsed.GetExpr(), parsed.GetSourceInfo(), "")
+	i := interpreter.NewInterpretable(prg)
+	res, _ := i.Eval(NewActivation(map[string]interface{}{}))
```

Comment:
Do we need to check if there is any error here?

Reply:
The eval returns a 'state' object that gives observed expression values during execution, that's what I'm ignoring here.